calculator-app/src/Components/Calculator.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import Results from './Results'
+import * as math from "mathjs"
+class Calculator extends React.Component {
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            display: 0,
+            allClear: 0,
+            Operators: null,
+            nextDisplay: null,
+            nextValue:false
+            
+
+        }
+    }
+    buttonValue(event){
+          let newValue = event.target.value
```

Comment:
every new  number value replaces the only input value, this means any number above 9 wont work.

Try to think of a way to keep adding on the numbers instead.